src/curated-corpus/helpers/helperFunctions.ts

```diff
@@ -55,6 +53,14 @@ export const transformProspectToApprovedItem = (
 export const transformUrlMetaDataToProspect = (
   metadata: UrlMetadata
 ): Prospect => {
+  // set language to undefined if metadata.language is an empty string or undefined.
+  // if not, then map it from string to its corresponding CorpusLanguage enum value
+  const language = !metadata.language
+    ? undefined
+    : metadata.language === 'en'
+    ? CorpusLanguage.En
+    : CorpusLanguage.De;
```

Comment:
This might be too confusing, I can break it down using an `if()` 😬

Reply:
Maybe a comment? In a follow-up PR perhaps as I'd really like to merge this one and make it easier for myself in the next frontend branch :)